python/py_pkg_cc_deps.bzl

```diff
@@ -83,30 +83,29 @@ repository named `tflm_pip_deps` via @rules_python in the WORKSPACE:
     )
 ```
 
-2. Use the repository rule `tflm_py_cc_headers` in the WORKSPACE to create an
-external repository with a target `@numpy_headers:numpy_headers`, passing the
-py_library target from @tflm_pip_deps obtained via requirement() and an
+2. Use the repository rule `py_pkg_cc_deps` in the WORKSPACE to create an
+external repository with a target `@numpy_cc_deps//:cc_headers`, passing the
```

Comment:
Does this work `@numpy_cc_deps:cc_headers`?

Reply:
No, the `//` is required when referring an external repository in a label—per the [Bazel docs on labels](https://bazel.build/concepts/labels) and as verified by trying it out.